index.js

```diff
@@ -51,7 +51,7 @@ module.exports = function(options) {
   return function cors(ctx, next) {
     // If the Origin header is not present terminate this set of steps.
     // The request is outside the scope of this specification.
-    const requestOrigin = ctx.get('Origin');
+    const requestOrigin = ctx.get('Origin') || ctx.origin;
```

Comment:
ctx.origin points to the url to which the client connects to not the origin header right? Is this fix correct?